doc/commands/module.8.rst

```diff
@@ -28,13 +25,18 @@
 Synopsis
 ========
 
-``dnf5 module <subcommand> [options] [<module-spec>]``
+``dnf5 module <subcommand> [options] [<module-spec>..]``
```

Comment:
There are 3 dots everywhere else (well, for `<package-spec>`, but I'd say `<module-spec>` is similar enough :D)